spec/directions/move_title_text_into_span_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::Directions::MoveTitleTextIntoSpan do
+
+  let(:element_1) do
+    new_element(
+      <<~HTML
+        <h1 data-type="document-title">foo</h1>
+      HTML
+    )
+  end
+
+  it 'calls v1' do
```

Comment:
it is a little dumb but I often say `it 'works'` for a spec with one test that is testing everything.  Here you are testing more than it just calling v1, you are testing that when you call v1 it does what you want.

Reply:
Glad to hear there's something as straightforward as `it 'works'`, I was honestly kind of stumped for what to put there 😆